systemtests/src/main/java/io/enmasse/systemtest/BrokerManagement.java

```diff
@@ -0,0 +1,17 @@
+package io.enmasse.systemtest;
```

Comment:
I think you can just merge this with ArtemisManagement as it doesn't really add anything by being in a separate  class.

Reply:
I would like to implement logic for apache activemq broker in the future (not sure if this broker will be used in enmasse)

Reply:
It's not a big deal, but me personal preference is that such a refactoring should be done when its needed. I'm fine with the current approach.